src/com/intellij/idea/plugin/hybris/tools/remote/http/AbstractHybrisHacHttpClient.java

```diff
@@ -111,22 +111,50 @@ public String login(@NotNull Project project, @NotNull HybrisRemoteConnectionSet
     }
 
     @NotNull
-    public final HttpResponse post(@NotNull Project project, @NotNull String actionUrl, @NotNull List<BasicNameValuePair> params, boolean canReLoginIfNeeded) {
+    public final HttpResponse post(
+        @NotNull Project project,
+        @NotNull String actionUrl,
+        @NotNull List<BasicNameValuePair> params,
+        boolean canReLoginIfNeeded,
+        final long timeout
+    ) {
+        return post(project, actionUrl, params, canReLoginIfNeeded, 6000L);
+    }
+
+    @NotNull
+    public final HttpResponse post(
+        @NotNull Project project,
+        @NotNull String actionUrl,
+        @NotNull List<BasicNameValuePair> params,
+        boolean canReLoginIfNeeded
+    ) {
+        return post(project, actionUrl, params, canReLoginIfNeeded, 6000L, null);
```

Comment:
I think that we should replace `6000L` to` timeout `

Reply:
done